### ISSUE TITLE
Add Learning Content

### ISSUE THREAD

**Issue by CyberChristy919** [PowerShell/PowerShellGallery#256]
### Summary of the new feature / enhancement

I would like to learn why Microsoft set PowerShell to untrusted and the steps to set it as trusted.

### Proposed technical implementation details (optional)

This is my understanding of why the PowerShell Gallery download is untrusted.

Microsoft PowerShell is hosted on a community repository that may contain harmful code. It's your responsibility to assess whether you trust the Gallery.

**Comment by alerickson** [comment]
@CyberChristy919 yes, that's correct.  Most companies that publish packages to the PowerShell Gallery sign their code with certificates, however we allow anyone to publish to the Gallery.  That said, it's against policy to publish anything malicious and we do take down those packages, we just can't guarantee that everything on the site is safe to download.
